core/tests/unit/kmx/kmx.cpp

```diff
@@ -241,13 +254,12 @@ run_test(const km::core::path &source, const km::core::path &compiled) {
       assert(ci->type != KM_CORE_CT_END && test_ci != test_context.end()); // Verify that both lists are same length
       assert(test_ci->type == ci->type && test_ci->marker == ci->marker);
     }
-
     km_core_context_items_dispose(citems);
-    if (text_store != buf) {
-      std::cerr << "text store has diverged from buf" << std::endl;
+    if ((!context_invalidated) && (text_store != core_context_str)) {
+      std::cerr << "text store has unexpectedly diverged from core_context" << std::endl;
       std::cerr << "text store: " << string_to_hex(text_store) << " [" << text_store << "]" << std::endl;
-      std::cerr << "context   : " << string_to_hex(buf) << " [" << buf << "]" << std::endl;
-      assert(false);
+      std::cerr << "core context   : " << string_to_hex(core_context_str) << " [" << core_context_str << "]" << std::endl;
```

Comment:
```suggestion
      std::cerr << "core context: " << string_to_hex(core_context_str) << " [" << core_context_str << "]" << std::endl;
```
...to align the output